### PR TITLE
Some grammar and capitalization error

### PR DESCRIPTION
**Pull request by lovetodevelop**
### Checklist ###

* [x] New tests have been added to show the fix or feature works
* [x] Grunt build and unit tests pass locally with these changes
* [x] If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com

### CODE REVIEW

**Review by kuohuanhuan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -87,8 +87,8 @@ Some example modules that can be excluded are:
 - **ajax/xhr**: The XMLHTTPRequest AJAX transport only.
 - **ajax/script**: The `<script>` AJAX transport only; used to retrieve scripts.
 - **ajax/jsonp**: The JSONP AJAX transport only; depends on the ajax/script transport.
-- **css**: The `.css()` method. Also removes **all** modules depending on css (including **effects**, **dimensions**, and **offset**).
-- **css/showHide**:  Non-animated `.show()`, `.hide()` and `.toggle()`; can be excluded if you use classes or explicit `.css()` calls to set the `display` property. Also removes the **effects** module.
+- **CSS**: The `.css()` method. Also removes **all** modules depending on css (including **effects**, **dimensions**, and **offset**).
```
Comment (mgol) [inline README.md:90]: This is not a correct change.

File: README.md [hunk 2]
```diff
@@ -359,11 +359,11 @@ testIframe( testName, fileName,
   } );
 ```
 
-This loads a page, constructing a url with fileName `"./data/" + fileName`.
+This loads a page, constructing a URL with fileName `"./data/" + fileName`.
 The iframed page determines when the callback occurs in the test by
 including the "/test/data/iframeTest.js" script and calling
 `startIframeTest( [ additional args ] )` when appropriate. Often this
-will be after either document ready or `window.onload` fires.
+will be after either the document is ready or `window.onload` fires.
```
Comment (mgol) [inline README.md:366]: `document ready` is a jQuery term which means a concrete thing, `the document is ready` doesn't seem to carry a similar meaning.

File: README.md [hunk 3]
```diff
@@ -87,8 +87,8 @@ Some example modules that can be excluded are:
 - **ajax/xhr**: The XMLHTTPRequest AJAX transport only.
 - **ajax/script**: The `<script>` AJAX transport only; used to retrieve scripts.
 - **ajax/jsonp**: The JSONP AJAX transport only; depends on the ajax/script transport.
-- **css**: The `.css()` method. Also removes **all** modules depending on css (including **effects**, **dimensions**, and **offset**).
-- **css/showHide**:  Non-animated `.show()`, `.hide()` and `.toggle()`; can be excluded if you use classes or explicit `.css()` calls to set the `display` property. Also removes the **effects** module.
+- **CSS**: The `.css()` method. Also removes **all** modules depending on css (including **effects**, **dimensions**, and **offset**).
+- **CSS/showHide**:  Non-animated `.show()`, `.hide()` and `.toggle()`; can be excluded if you use classes or explicit `.css()` calls to set the `display` property. Also removes the **effects** module.
```
Comment (mgol) [inline README.md:91]: Not a correct change, the name should stay lowercase.

**Comment by timmywil** [comment]
Thanks for your contribution, but most of these changes are not necessary. Some of them we may get to while addressing other issues.

**Pull request closed without merging**